diesel/src/query_builder/select_statement/mod.rs

```diff
@@ -47,6 +49,7 @@ pub struct SelectStatement<
     LimitOffset = LimitOffsetClause<NoLimitClause, NoOffsetClause>,
     GroupBy = NoGroupByClause,
     Locking = NoLockingClause,
+    Having = NoHavingClause,
```

Comment:
As said above, let's move this up before `Locking`

Reply:
Fixed.